next-tavla/app/(admin)/boards/components/Column/Delete.tsx

```diff
@@ -38,14 +38,14 @@ function Delete({ board }: { board: TBoard }) {
                     <CloseIcon />
                 </SecondarySquareButton>
                 <Image src={sheep} alt="" className="h-50 w-50" />
-                <Heading1 className="text-rem-4">Slett tavle</Heading1>
-                <LeadParagraph>
+                <Heading1 className="text-rem-4 m-0">Slett tavle</Heading1>
```

Comment:
Kunne vi brukt Heading2 her? Da hadde vi sluppet å bruke egne klasser her

Reply:
Takk for innspill 😄 byttet til Heading2